servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered.glsl

```diff
@@ -352,7 +352,7 @@ void vertex_shader(vec3 vertex_input,
 	}
 
 #ifdef OVERRIDE_POSITION
-	vec4 position;
+	vec4 position = vec4(vertex, 1.0);
```

Comment:
```suggestion
	vec4 position = vec4(1.0);
```

We should just default to 1.0. In the docs, we tell people that this is an output value only (i.e. they shouldn't read from it). This point is in clip space anyway, so `vertex` isn't any better than 0 or 1.

Reply:
Makes sense!